src/components/InlineAddButton.jsx

```diff
@@ -83,6 +84,9 @@ export class InlineAddButton extends PureComponent {
 
   async handleClick(e) {
     e.stopPropagation();
+    if (this.state.inputValue.trim() === '') {
+      return this.setState({ inputValue: '', status: 'initial' });
+    }
```

Comment:
Hvis vi har lyst til å bruke tidligere state når vi oppdaterer state så burde vi ta i bruk variablene til funksjonen vi sender inn. React kan gjøre optimaliseringer som gjør at `this.state` ikke alltid er oppdatert.
Ála `this.setState(prevState => {...})`

Reply:
Tenker du noe ala sånn:

```suggestion
    if (this.state.inputValue.trim() === '') {
      return this.setState({ inputValue: '', status: 'initial' });
    }
```this.setState(prevState => ({inputValue:prevState.inputValue, status:prevState.status}));

For om det er det du tenkte så vil det ikke fungere helt. If'en brukes for å sikre at det ikke skrives inn tomme strenger i feltet og det blir addet, samt. så inneholder prevstate akkurat det samme som det this.state gjør.